package.json

```diff
@@ -12,8 +12,8 @@
     "format:check": "prettier --list-different \"**/*.{js,md}\"",
     "build:es": "cross-env BABEL_ENV=es babel src --out-dir es",
     "build:commonjs": "cross-env BABEL_ENV=commonjs babel src --out-dir lib",
-    "build:umd": "cross-env NODE_ENV=development webpack",
-    "build:umd:min": "cross-env NODE_ENV=production webpack",
+    "build:umd": "cross-env NODE_ENV=development webpack && es-check es5 dist/redux-actions.js",
+    "build:umd:min": "cross-env NODE_ENV=production webpack && es-check es5 dist/redux-actions.min.js",
```

Comment:
I confirmed that this failed before my change (not it passed on the unminified before due to eval sourcemaps obscuring the actual code)